resources/sass/base/_helpers.scss

```diff
@@ -56,6 +56,16 @@
     margin: 0px 1em;
 }
 
+.inline-list li {
```

Comment:
maybe add some `padding-right`?

Reply:
I think too much padding looks weird, but I'll add a few px for your sake ;)